src/App.js

```diff
@@ -0,0 +1,109 @@
+import { Component } from 'react';
+import AppHeader from './AppHeader';
+import ItemsList from './ItemsList';
+import AppFooter from './AppFooter';
+
+class App extends Component {
+    constructor() {
+        super()
+        this.state = {
+            inputfield: '',
+            inputs: [],
```

Comment:
the same here about avoiding referring DOM in the state names. Preferred name can be `toDoItems`

Reply:
Sure, changed `inputs` to `toDoItems`.